src/components/card/Description.vue

```diff
@@ -212,18 +224,35 @@ export default {
 		showAttachmentModal() {
 			this.modalShow = true
 		},
-		addAttachment(attachment) {
-			const descString = this.$refs.markdownEditor.easymde.value()
+		async addAttachment(attachment) {
 			let embed = ''
 			if ((attachment.type === 'file' && attachment.extendedData.hasPreview) || attachment.extendedData.mimetype.includes('image')) {
 				embed = '!'
 			}
 			const attachmentString = embed + '[📎 ' + attachment.data + '](' + this.attachmentPreview(attachment) + ')'
-			const newContent = descString + '\n' + attachmentString
+			const cursor = this.$refs.markdownEditor.easymde.codemirror.getCursor()
+			const descStringLines = this.$refs.markdownEditor.easymde.value().split('\n')
+			let lineUpatedLength = 0
+
+			for (let i = 0; i < descStringLines.length; i++) {
+				if (i === cursor.line) {
```

Comment:
Using a for-loop here seems excessive, if you only want to change one line. How anout this?

```js
if (descStringLines[cursor.line]) {
        const line = descStringLines[cursor.line]
	descStringLines[cursor.line] = line.substring(0, cursor.ch) + attachmentString + line.substring(cursor.ch)
	lineUpatedLength = descStringLines[cursor.line].length
}
```

Reply:
@marcelklehr it should work. I've changed this way.